lessons/operator/01/README.md

```diff
@@ -1,122 +1,121 @@
+Operators can add new components, modify the behavior of filters and handlers, silence noisy streams of data, or add new database sinks *at any time*, without the need to re-deploy.
 
+Components are defined in YAML files, which are pushed to the API to build the pipeline.
+These YAML files can be stored in source-control alongside your application code, and can be tested, versioned, and managed like any other code file.
+This workflow takes the burden off of system operators, by allowing engineers to self-service their monitoring configurations, with security and access controls at every layer.
 
-## Use cases
+To learn more, please check out our [monitoring as code whitepaper](https://sensu.io/resources/whitepaper/monitoring-as-code-with-sensu).
 
-Sensu's observability pipeline
+### Bonsai Asset Repository
 
-- **Eliminate data silos**
+Another essential component of Sensu is the [Bonsai Asset Repository](https://bonsai.sensu.io/).
+This service delivers binary assets directly to the agents and the backend, based on the configurations defined in YAML.
+Operators and developers can focus only on the YAML configuration, without worrying about how to securely deliver new executables to the edge.
 
-  By decoupling data collection from storage and analysis, Sensu enables you to consolidate data into data platforms like Elasticsearch, Splunk, and Sumo Logic.
+There are hundreds of open-source plugins available on Bonsai, supporting nearly every imaginable integration. 
+[Have a look around](https://bonsai.sensu.io/assets)! 
+If what you need isn't already available, the [Sensu Plugin SDK](https://github.com/sensu/sensu-plugin-sdk) makes it easy to develop and publish new custom plugins.
 
-- **Fill gaps between existing monitoring and observability tools**
+## Discussion
 
-  No one monitoring or observabilty tool can do it all, yet the proliferation of tools can in itself be a burden.
-  Adopting an observability pipeline eliminates the risks typically inherent to tool sprawl (e.g. data silos).
+In this lesson we covered the basics of Sensu's architecture, data model, and monitoring-as-code workflow. 
+The rest of the workshop is a series of hands-on exercises that walk you through some of the essential features of Sensu, in a sandboxed environment.
 
-- **Consolidate monitoring tools**
+### All You Have to Think About is `sensuctl` and YAML
 
-  Sensu supports your existing monitoring plugins and exporters – including Nagios-style monitoring scripts, StatsD, Telegraf, Prometheus – as well as hundreds of Sensu native integrations.
+The description above might seem somewhat complicated. 
+But the beauty of Sensu is that these features are built into the platform, so you don't have to worry about them. 
 
-- **Automate diagnosis & self-healing**
+As an operator, the majority of your time is spent editing YAML, and running high-level commands from a command-line shell.
```

Comment:
insert "interacting with Sensu" in "the majority of your time is spent" ??? 

"the majority of your time _interacting with Sensu_ is spent..."